include/glow/Base/Image.h

```diff
@@ -93,20 +93,51 @@ bool writePngImage(Tensor *T, const char *filename,
                    llvm::ArrayRef<float> mean = zeroMean,
                    llvm::ArrayRef<float> stddev = oneStd);
 
-/// Read a png image and preprocess it according to several parameters.
-/// \param filename the png file to read.
-/// \param imageNormMode normalize values to this range.
-/// \param imageChannelOrder the order of color channels.
+/// A helper function called by readPngImageAndPreprocess that does all the
+/// reading file and preprocessing work.
+/// \param imageData the tensor into which the preprocessed image data will be
+/// stored. \param filename the png file to read. \param imageNormMode normalize
```

Comment:
Bad formatting of the comment. Each `\param` should be at the beginning of a line.